lms/djangoapps/commerce/management/commands/configure_commerce.py

```diff
@@ -0,0 +1,72 @@
+"""
+Enable or disable commerce configuration.
```

Comment:
Over time the purpose of this management command will expand beyond enable/disable, so I think this docstring should be more generic.